tests/data/test_dataset_access.py

```diff
@@ -304,7 +304,7 @@ class TestLoadHelpers:
                     "HeH": {"STO-3G": ["0.50"]},
                 },
                 [["full"], ["full"], ["0.48", "0.50"]],
```

Comment:
Could we have a non-full unit-test as well?

Reply:
yep, on it